### DISCUSSION TITLE
Deepgram model

### DISCUSSION THREAD

**Discussion by faizansahib** [deepgram/community#385] [General help] [answered]
which machine learning model  is used by deepgram?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @faizansahib, I'd invite you to check out our blog post on our latest model release: [Introducing Nova-2: The Fastest, Most Accurate Speech-to-Text API](https://deepgram.com/learn/nova-2-speech-to-text-api)

In the post, we share some details around our transformer-based architecture, and how we've leveraged our architecture as well as data curation and training methodology to create an extremely fast and accurate model.
